.circleci/build-verilator.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# build verilator
+
+# turn echo on and error on earliest command
+set -ex
+
+cd $HOME/project
+
+cd sims/verisim
+
+if [ ! -d "$HOME/project/sims/verisim/verilator" ]; then
+    # make verilator
+    make verilator_install
```

Comment:
I really enjoy the multiple levels of nesting going on here. CircleCI reimplements make, which calls a script who's only task is to call make.

Reply:
Wrapper on a wrapper on a wrapper